main.go

```diff
@@ -229,21 +229,28 @@ func (a *Agent) List() ([]*agent.Key, error) {
 		return nil, fmt.Errorf("could not reach YubiKey: %w", err)
 	}
 
-	pk, err := getPublicKey(a.yk, piv.SlotAuthentication)
-	if err != nil {
-		return nil, err
+	var keys []*agent.Key
+	for _, slot := range slots {
+		var pk ssh.PublicKey
+		var err error
+		pk, err = getPublicKey(a.yk, slot)
+		if err != nil {
+			continue
```

Comment:
Here we're ignoring the errors silently - not sure if it's correct. One option would be to return error on 9a slot but ignore errors on others like before. Comments welcome.

From my point of view, it makes sense to check other slots at least in the expected "not found" error; we could also just log the error but load all slots in any case.